en/docs/guides/login/configure-ws-federation-single-sign-on.md

```diff
@@ -31,17 +31,17 @@ To configure WS-Federation SSO:
 
```

Comment:
the file above this (configure-recaptcha-for-single-sign-on.md) has been renamed. Is this by accident?

Reply:
No, tried to make things consistent with a verb in the beginning.